src/identify_client.cpp

```diff
@@ -239,7 +239,7 @@ namespace
 
 	struct generic_map_entry
 	{
-		int offset;
+		int const offset;
```

Comment:
this is a bit inconsistent. all fields should really be const, but iirc, the array that these objects are in is already const, so I think this is redundant